docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
+
+* Python 3,
+* Neptune account |register|,
+* ``pytorch-lightning==0.9.0`` and ``torchvision`` installed (lightning |lightning-install|),
+* Minimal familiarity with the PyTorch Lightning.
+
+Installation
+^^^^^^^^^^^^
+**Install neptune-client**
+
+From PyPI:
+
+.. code-block:: bash
+
+    pip install neptune-client -U
+
+From Conda:
 
 .. code-block:: bash
 
-    pip install neptune-client
+    conda install neptune-client -c conda-forge
+
+Now, you have all dependencies installed. Let's move to the actual integration.
+
+Step 1: Import Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^
+Import necessary libraries.
 
+.. code-block:: python3
+
+    import os
+
+    import torch
+    from torch.nn import functional as F
+    from torch.utils.data import DataLoader
+    from torchvision.datasets import MNIST
+    from torchvision import transforms
+
+    import pytorch_lightning as pl
 
-Create the **LightningModule**
-------------------------------
+Notice ``pytorch_lightning`` at the bottom.
 
-Nothing changes here, just your typical ``LightningModule`` structure.
+Step 2: Define Hyper-Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Define hyper-parameters for model training.
 
 .. code-block:: python3
 
-    class CoolSystem(pl.LightningModule):
+    PARAMS = {'max_epochs': 3,
+              'learning_rate': 0.005,
+              'batch_size': 32}
+
+Hyper-parameters are passed in the regular Python dictionary. You will see them logged in Neptune parameters tab.
+
+Step 3: Define LightningModule and DataLoader
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Implement minimal example of the ``pl.LightningModule`` and simple ``DataLoader``.
+
+.. code-block:: python3
 
+    # pl.LightningModule
+    class LitModel(pl.LightningModule):
         def __init__(self):
-            ...
+            super().__init__()
+            self.l1 = torch.nn.Linear(28 * 28, 10)
 
         def forward(self, x):
-            ...
+            return torch.relu(self.l1(x.view(x.size(0), -1)))
 
         def training_step(self, batch, batch_idx):
-            ...
+            x, y = batch
+            y_hat = self(x)
+            loss = F.cross_entropy(y_hat, y)
+            result = pl.TrainResult(minimize=loss)
+            result.log('train_loss', loss)
+            return result
 
         def configure_optimizers(self):
-            ...
+            return torch.optim.Adam(self.parameters(), lr=PARAMS['learning_rate'])
+
+    # DataLoader
+    train_loader = DataLoader(MNIST(os.getcwd(), download=True, transform=transforms.ToTensor()),
+                              batch_size=PARAMS['batch_size'])
 
-        @pl.data_loader
-        def train_dataloader(self):
-            ...
+Cross Entropy loss in the ``training_step`` method will be logged to Neptune in every training step. ``DataLoader`` is a purely PyTorch object. Notice, that you pass ``learning_rate`` and ``batch_size`` from the ``PARAMS`` dictionary - all params will be logged as experiment parameters.
```

Comment:
I think extracting this line here and presenting it could be useful to focus the reader. 

```python
result.log('train_loss', loss)
```

Reply:
added.